src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap

```diff
@@ -8423,6 +8423,10 @@ If the provided project does not exist, the list of events will be empty.",
       "description": "Create, update, delete, and manage [segments](https://docs.getunleash.io/reference/segments).",
       "name": "Segments",
     },
+    {
+      "description": "Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts) enabling apps to access Unleash API programatically.",
+      "name": "Service Accounts",
+    },
```

Comment:
If we're updating the tag description we should update the snap as well.

Reply:
yes!